getting-started/multiband-analysis.rst

```diff
@@ -0,0 +1,293 @@
+See :doc:`/install/setup` for details on doing this.
+
+As in :doc:`part 3 <display>`, you'll be working inside an interactive Python session for this tutorial.
+You can use the default Python shell (:command:`python`), the `IPython shell`_, or even run from a `Jupyter Notebook`_.
+Ensure that this Python session is running from the shell where you ran :command:`setup lsst_distrib`.
+
+Loading forced photometry measurement catalogs with the Butler
+==============================================================
+
+The :command:`forcedPhotCoadd.py` command-line task (:ref:`getting-started-tutorial-forced-coadds`) created ``deepCoadd_forced_src`` datasets for each coadd in the example data repository.
+Being forced photometry catalogs, rows in each ``deepCoadd_forced_src`` table correspond row-for-row across all coadds in different filters for a sky map patch.
+Since you don't have to do additional cross-matching, these ``deepCoadd_forced_src`` datasets are convenient.
+
+To get these datasets, open a Python session (`IPython`_ or `Jupyter Notebook`_) and create a Butler:
+
+.. code-block:: python
+
+   import lsst.daf.persistence as dafPersist
+   butler = dafPersist.Butler(inputs='DATA/rerun/coaddForcedPhot')
+
+This Butler is using the ``coaddForcedPhot`` rerun you created for the :command:`forcePhotCoadd.py` command-line task's outputs.
+
+Next, use the Butler to get the ``deepCoadd_forced_src`` datasets for both filters:
+
+.. code-block:: python
+
+   rSources = butler.get('deepCoadd_forced_src', {'filter': 'HSC-R', 'tract': 0, 'patch': '1,1'})
+   iSources = butler.get('deepCoadd_forced_src', {'filter': 'HSC-I', 'tract': 0, 'patch': '1,1'})
+
+These datasets correspond to the central patch (``1,1``) coadd for both the HSC-R and HSC-I filters.
+There are other patches in the sky map, but this tutorial will focus on the central patch.
+
+Getting calibrated PSF photometry
+=================================
+
+The ``base_PsfFlux_flux`` column of these ``deepCoadd_forced_src`` datasets is the flux from the linear least-squares fit of the PSF model to the source.
+From the source table's schema you know this flux has units of counts:
+
+.. code-block:: python
+
+   iSources.getSchema().find('base_PsfFlux_flux').field.getUnits()
+
+Transforming this flux into a magnitude requires knowing the coadd's zeropoint.
+We'll get this metadata from the coadd dataset itself.
+The coadd we made in :doc:`part 3 <coaddition>` with :command:`assembleCoadd.py` doesn't have calibration info attached to it, though.
+Instead, we want the ``deepCoadd_calexp`` dataset that was created by the :command:`detectCoaddSources.py` command-line task.
+This ``deepCoadd_calexp`` is a coadd that includes calibration information attached by source detection tasks.
+
+First, load the dataset for both filters:
+
+.. code-block:: python
+
+   rCoaddCalexp = butler.get('deepCoadd_calexp',  {'filter': 'HSC-R', 'tract': 0, 'patch': '1,1'})
```

Comment:
I think you can get the calib directly using `'deepCoadd_calexp_calib'` as the dataset type.  I haven't checked, and if you'd prefer not, it's not that big of a deal to just leave as is.